libs/search/src/lib/facets/facets.service.spec.ts

```diff
@@ -7,11 +7,16 @@ import { FacetsService } from './facets.service'
 let requestAggregations
 let responseAggregations
 
+const logServiceMock = {
+  warn: jest.fn(),
+  log: jest.fn(),
+  error: jest.fn(),
+}
 describe('FacetsService', () => {
   let service: FacetsService
 
   beforeEach(() => {
-    service = new FacetsService()
+    service = new FacetsService(logServiceMock)
```

Comment:
Is there a reason for not using test bed here ?

Reply:
It's slower and useless.